src/assets/scss/layouts/_technologies.scss

```diff
@@ -0,0 +1,24 @@
+.technologies {
+  width: 100%;
+  height: 80vh;
+  height: calc(var(--vh, 1vh) * 80);
+  overflow: hidden !important;
+  background-color: $background-light;
+  padding: 3rem 8rem;
+
+  &__title {
+    h1 {
+      @include font;
+      color: $font-color;
+      font-size: 3rem;
+      letter-spacing: 0.12em;
+      text-transform: uppercase;
+    }
+  }

```

Comment:
*since you are using BEM, you can remove the h1 and only have 

```
&__title {
      @include font;
      color: $font-color;
      font-size: 3rem;
      letter-spacing: 0.12em;
      text-transform: uppercase;
  }
```

Reply:
other thing, if you want to center your title you can use `text-align: center`

This is only an extra info, you don't need to do this if you don't want to 😆 

<img src="https://user-images.githubusercontent.com/50960425/105024101-9ab82480-5a43-11eb-8585-325ca355f045.png" width="320"/>
<img src="https://user-images.githubusercontent.com/50960425/105024108-9d1a7e80-5a43-11eb-923e-6bda475e9b58.png" width="320"/>

Reply:
done 🍇